master/internal/config/resource_manager_config.go

```diff
@@ -152,7 +152,11 @@ func (a AgentResourceManagerConfig) Validate() []error {
 
 // KubernetesResourceManagerConfig hosts configuration fields for the kubernetes resource manager.
 type KubernetesResourceManagerConfig struct {
+	// deprecated, no longer in use.
 	Namespace string `json:"namespace"`
```

Comment:
note: would it be worth creating a tech debt ticket in the backlog to clean up this feature from the repo two months from now?

Reply:
Yeah, thats a good idea. I'll make one